nfd-daemonset-master-topologyupdater-combined.yaml.template

```diff
@@ -1,120 +0,0 @@
-# This template contains an example of running nfd-master and nfd-worker in the
```

Comment:
This was supposed to be a manifest template or deploying nfd-master and topology updater in a single pod

Reply:
I think we should keep this manifest template too

Reply:
done